### PR TITLE
Enable loading of external assets through a "twiddle.json"

### PR DESCRIPTION
**Pull request by Gaurav0**
- Fixes issue #38 as well; allows you to load any version of ember or ember-data

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -73,5 +75,10 @@ module.exports = function() {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
-  return app.toTree();
+  var bowerTree = funnel('bower_components', {
```
Comment (joostdevries) [inline ember-cli-build.js]: could you make sure this gets fingerprinted + minified to a single file?

File: app/gist/new/route.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ export default GistRoute.extend({
       filePath: 'router.js',
       content: 'import Ember from \'ember\';\nvar Router = Ember.Router.extend({\n  location: \'none\'\n});\n\nRouter.map(function() {\n});\n\nexport default Router;\n'
     }));
+    model.get('files').pushObject(this.store.createRecord('gistFile', {
+      filePath: 'twiddle.json',
+      content: '{\n  "dependencies": {\n    "jquery": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.js",\n    "ember": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/1.13.5/ember.js",\n    "ember-data": "https://cdnjs.cloudflare.com/ajax/libs/ember-data.js/1.13.5/ember-data.js"\n  }\n}'
```
Comment (joostdevries) [inline app/gist/new/route.js]: specific reason you're not using builds.emberjs.com?

**Comment by joostdevries** [comment]
Like where this is going but I'm pondering over doing a client-side 'build' (XHR'ing all the vendor js once and injecting a bigass js string) vs. injecting multiple script src tags.

File: ember-cli-build.js (hunk 1 above)
Comment (Gaurav0) [inline ember-cli-build.js]: Not sure how to both do that and make it possible for the files to be overridden. Any ideas?

File: app/gist/new/route.js (hunk 2 above)
Comment (Gaurav0) [inline app/gist/new/route.js]: The CSP was complaining that builds.emberjs.com was not https

Comment (Gaurav0) [inline app/gist/new/route.js]: Besides, I'm not sure that builds.emberjs.com is meant to be used as a CDN

**Comment by Gaurav0** [comment]
Perhaps I am misunderstanding, but can you even do that with external dependencies?

**Comment by joostdevries** [comment]
probably not easily at least.

File: app/gist/new/route.js (hunk 2 above)
Comment (stefanpenner) [inline app/gist/new/route.js]: maybe we should get builds to serve over HTTPS

Comment (joostdevries) [inline app/gist/new/route.js]: :-)

Comment (joostdevries) [inline app/gist/new/route.js]: @Gaurav0 the current link is fine.

**Comment by joostdevries** [comment]
@Gaurav0 I'll merge this in but a deploy will be later this week (but before the Ember NYC meetup I promise ;-)). Want to try some stuff out first

**Comment by Gaurav0** [comment]
@joostdevries Thank you.

**Comment by joostdevries** [comment]
hoped to deploy tonight but had a minor crisis to attend :-(

**Comment by stefanpenner** [comment]
> hoped to deploy tonight but had a minor crisis to attend :-(

hope all is well :)

**Pull request merged**